_posts/2016-10-01-hacktober-thumbs-high-level-dumplings-tgif-101.md

```diff
@@ -0,0 +1,94 @@
+---
+
+title: 'Hacktober thumbs high level dumplings: TGIF (101)'
+author: Jenn
+layout: blog-post-tgif
+comments: false
+
+---
+
+
+TGIF! – Thank Grohl, Gauss, Glob, Galileo and Science it’s Friday-adjacent (okay Saturday), and again it’s time for our weekly reading recommendations, prepared to help you get some good reading time on your weekend. As usual, they’re curated for you by [Jenn](http://twitter.com/jennwrites). Enjoy!
+
+## Open Source
+
+[Hacktoberfest is open to everyone in the global community!](https://hacktoberfest.digitalocean.com/) DigitalOcean and GitHub team up to celebrate and support open source.
+
+[Today we met to talk about Open Source Communities and editorial contributors, and how to bring both together.](https://twitter.com/hoodiehq/status/781372613985652736)
+
+[Yahoo open sources its porn-detecting neural network](https://techcrunch.com/2016/09/30/yahoo-open-sources-its-porn-detecting-neural-network/)
+
+
+## Design
+
+[The Thumb Zone: Designing For Mobile Users](https://www.smashingmagazine.com/2016/09/the-thumb-zone-designing-for-mobile-users/)
+
+> Different languages and cultures require different design considerations for typography and copy. This agency in Japan explains nuances of Japanese type. <cite>[Seven rules for perfect Japanese typography](https://medium.com/aq-writes/seven-rules-for-perfect-japanese-typography-c377fbf49d5#.bh7salqsc)</cite>
+
+[Design against Distraction to Make Your Life Better](https://thestand.investec.co.uk/calm-technology-design-distraction-make-life-better/)
+
+[Emoji outlines](https://twitter.com/rauchg/status/781590245720494080)
+
+## The Web and Development
+
+[YouTube’s new “offline first” product for India treats telcos as damage and routes around them](http://boingboing.net/2016/09/28/youtubes-new-offline-first.html)
+
+[New Employee: Can you give me a high level overview of how the system works?](https://twitter.com/moonpolysoft/status/781868129269919744)
+
+[Better living through documentation](https://datamade.us/blog/better-living-through-documentation)
+
+
+## The Tech World and Culture
+
+[This Old-Ass Commodore 64 Is Still Being Used to Run an Auto Shop in Poland](http://sploid.gizmodo.com/this-old-ass-commodore-64-is-still-being-used-to-run-an-1787196319)
+
+[This app fuels the illicit trade of dumplings and pork knuckles in Southern California](http://www.latimes.com/business/la-fi-wechat-food-20160926-snap-story.html)
+
+> I work in a book store, so I thought, I wonder if you can Scrum that? The answer is... potentially, yes. <cite>[Scrum at the Book Store](http://developingian.com/scrum-at-the-book-store/)</cite>
```

Comment:
@IanDCarroll hey look! your blog post is in the latest Hoodie TGIF newsletter & blog :)